Lib/tempfile.py

```diff
@@ -133,32 +133,18 @@ def _sanitize_params(prefix, suffix, dir):
     return prefix, suffix, dir, output_type
 
 
-class _RandomNameSequence:
-    """An instance of _RandomNameSequence generates an endless
-    sequence of unpredictable strings which can safely be incorporated
-    into file names.  Each string is six characters long.  Multiple
-    threads can safely use the same instance at the same time.
-
-    _RandomNameSequence is an iterator."""
+def _RandomNameSequence():
```

Comment:
When I touch the stdlib, I'm also shy and hesitate to touch anything. For this specific change, since it's a private function, I guess that it's ok to rename the function. Will do once I understand @serhiy-storchaka concern about thread safety.